### DISCUSSION TITLE
Feature Request: Credit Note in Invoice

### DISCUSSION THREAD

**Discussion by apkuki** [kimai/kimai#4329] [General] [answered]
Situation: My Client accidentally paid the same Invoice twice.

Therefore I wanted to add something like a "credit note" to the next invoice to account for that double payment. Sadly you cannot ad a manual entry with a minus position (=credit)
![image](https://github.com/kimai/kimai/assets/98207992/a0a51d8b-e976-40c1-994b-c1749f94ff45)

I solved it by adding a timesheet entry and then overwriting the amount directly in SQL database to -381.67
![image](https://github.com/kimai/kimai/assets/98207992/ea9e6feb-1718-4bd0-a2b8-c7e214e61242)

It worked fine and the system actually calculates everything correctly and the invoice is calculated correctly
![image](https://github.com/kimai/kimai/assets/98207992/1c79649a-2117-4945-afd9-7293d9e441a7)

It could also happen that I as a client make a mistake and accidentally invoiced too much in an earlier invoice

Therefore it would be nice to generally get the ability to add minus positions or manual credit note positions to the timesheet or invoice.

**Comment by kevinpapst** [comment, marked as answer]
https://www.kimai.org/store/expenses-bundle.html

**Reply by apkuki** [reply]
Did not see that it allows for negative expenses. Thanks
